src/lib/storage/vector_compression/compressed_vector_type.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <cstdint>
+#include <optional>
 
 #include <boost/version.hpp>
 #if BOOST_VERSION < 107100                 // TODO(anyone): remove this block once Ubuntu ships boost 1.71
```

Comment:
Just checked: Ubuntu 20 ships boost 1.71 (https://launchpad.net/ubuntu/focal/+package/libboost-dev). Do we want to support older Ubuntu versions? I'd say no.

Reply:
see https://github.com/hyrise/hyrise/pull/2522#discussion_r1091640241